doc/extending.rst

```diff
@@ -31,10 +31,10 @@ Normally `doit` tasks are defined in a `dodo.py` file.
 This file is loaded, and the list of tasks is created from
 the dict containing task meta-data from the *task-creator* functions.
 
-Subclass TaskLoader to create a custom loader:
```

Comment:
Since we are not even adding a deprecation warning (yet).
Can we just mention there is a "deprecated" older version?